### DISCUSSION TITLE
Cannot edit files

### DISCUSSION THREAD

**Discussion by nickchomey** [community/community#17011] [Other Feature Feedback, Questions, & Ideas]
Often when I click `Edit this file` when viewing a file, it opens and the file content is both blank and cannot be clicked upon to edit. I have to cancel and try again, refresh page, etc... sometimes in excess of 5 times. 

https://user-images.githubusercontent.com/88559987/169590359-cbc4cce1-a838-4438-aedc-5175d9521e35.mp4

I've tried clearing my browser cache etc... Nothing seems to help. I'm using MS Edge, if that helps.

Hopefully this is in the right place!

**Comment by dmilburn** [comment]
Hi from the Codespaces team! @nickchomey, I'm sorry you're experiencing this friction but I think this might be the wrong category unless I'm misunderstanding. It doesn't look like you're using the codespaces product for editing, is that correct? [General feedback](https://github.com/github/feedback/discussions/categories/general-feedback?discussions_q=is%3Aunanswered+created%3A%3E2022-05-12+category%3A%22General+Feedback%22) might be a better place for the right team to see this

Though [pressing the "." key](https://docs.github.com/en/codespaces/the-githubdev-web-based-editor) might unblock you for now!

**Reply by nickchomey** [reply]
Sorry! I've moved it to general now. 

I will give "." a try next time it happens!

**Reply by nickchomey** [reply]
Just happened again and "." didn't do anything.

**Reply by dmilburn** [reply]
Oh sorry i was unclear, I meant pressing "." on the page before you click the edit button. Thanks for moving it over!
